# time when command is sent
std_msgs/Header header 

# Set duration for which to record all incoming images on topic (in seconds)
float32 record_duration_s 